starter/.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "root": true,
```

Comment:
This file could be simplified since the rules are already defined in the eslint-config:

```json
{
  "root": true,
  "extends": ["@nationalbankbelgium"],
  "overrides": [
    {
      "files": ["*.ts"],
      "parserOptions": {
        "project": ["tsconfig.json", "e2e/tsconfig.json"],
        "createDefaultProgram": true
      },
      "rules": {
        "@angular-eslint/component-selector": [
          "error",
          {
            "prefix": "app",
            "style": "kebab-case",
            "type": "element"
          }
        ],
        "@angular-eslint/directive-selector": [
          "error",
          {
            "prefix": "app",
            "style": "camelCase",
            "type": "attribute"
          }
        ]
      }
    }
  ]
}
```